GitHubCore.ps1

```diff
@@ -217,7 +221,7 @@ function Invoke-GHRestMethod
             if ($PSCmdlet.ShouldProcess($jobName, "Start-Job"))
             {
                 [scriptblock]$scriptBlock = {
-                    param($Url, $Method, $Headers, $Body, $ValidBodyContainingRequestMethods, $TimeoutSec, $ScriptRootPath)
+                    param($Url, $Method, $Headers, $Body, $ValidBodyContainingRequestMethods, $TimeoutSec, $LogRequestBody, $ScriptRootPath)
```

Comment:
Out-of-scope for this PR, but instead of maintaining two codepaths, it seems easier to define one scriptblock and either invoke as job or invoke locally.  If there are differences in the two paths, we can separate out what is common and what is not.

Reply:
Good suggestion.  I've opened up #61 to track that suggestion as a future enhancement,